xapian-core/include/xapian/weight.h

```diff
@@ -449,25 +455,37 @@ class XAPIAN_VISIBILITY_DEFAULT TfIdfWeight : public Weight {
      *     @li 'p': Prob    idfn=log((N-Termfreq)/Termfreq)
      *     @li 'f': Freq    idfn=1/Termfreq
      *     @li 's': Squared idfn=log(N/Termfreq)^2
+     *     @li 'P': Pivoted idfn=log((N+1)/Termfreq)
      *
      * @li The third and the final character indicates the normalization for
      *     the document weight.  The following normalizations are currently
      *     supported:
      *
      *     @li 'n': None wtn=tfn*idfn
+     *     @li 'P': Pivoted wtn=wqf(tfn*idfn*(1-slope+(slope*normlen))+delta*idfn) where
+     *         parameters slope and delta values should be non-negative & non-zero when
+     *         using pivoted normalization string. Specifically, "PPP" normalization
+     *         string should be used to use Piv+ normalization. In addtion, normalization
+     *         string such as "Pxx", "xPx" and "xxP" can also be used to have more options
+     *         of combinations for wdfn, idfn and wtn normalizations respectively.
```

Comment:
I don't think it's clear enough that the `x` characters here are mean to be placeholders rather than literal - perhaps better to instead say something like "The different types of pivoted normalisation can also be used in combination with other normalisations"?